bciers/apps/reporting/src/app/components/products/ProductionData.tsx

```diff
@@ -0,0 +1,55 @@
+import ProductionDataForm from "./ProductionDataForm";
+import { buildProductionDataSchema } from "@reporting/src/data/jsonSchema/productionData";
+import { getProductionData } from "@bciers/actions/api";
+import { getOrderedActivities } from "../../utils/getOrderedActivities";
+import { getFacilitiesInformationTaskList } from "../taskList/2_facilitiesInformation";
+import { Suspense } from "react";
+
+interface Props {
+  report_version_id: number;
+  facility_id: string; // UUID
+}
+
+const ProductionData: React.FC<Props> = async ({
+  report_version_id,
+  facility_id,
+}) => {
+  const response = await getProductionData(report_version_id, facility_id);
+
+  const allowedProductNames = response.allowed_products.map((p) => p.name);
+  const allowedProducts = response.allowed_products.map((p) => ({
+    product_id: p.id,
+    product_name: p.name,
+  }));
+
+  const schema: any = buildProductionDataSchema(
+    "Jan 1",
+    "Dec 31",
+    allowedProductNames,
+  );
+
+  const orderedActivities = await getOrderedActivities(
+    report_version_id,
+    facility_id,
+  );
+  const taskListElements = await getFacilitiesInformationTaskList(
```

Comment:
getFacilitiesInformationTaskList does not return a promise, so we can consider removing await here 
  const taskListElements = getFacilitiesInformationTaskList(
    report_version_id,
    facility_id,
    orderedActivities,
  );

Reply:
A question: Shouldn't the selected Regulated Products from the operation review page display here? It currently only shows fixture data, or may be I am missing something, We can pair up on that to understand it better.